babel-watch.js

```diff
@@ -247,18 +246,21 @@ function restartApp() {
     }
   }
 
-  // Support for --debug option
   const runnerExecArgv = process.execArgv.slice();
-  if (program.debug) {
-    runnerExecArgv.push('--debug=' + program.debug);
-  }
-  // Support for --inspect option
-  if (program.inspect) {
-    runnerExecArgv.push('--inspect');
-  }
-  // Support for --debug-brk
-  if(program.debugBrk) {
-    runnerExecArgv.push('--debug-brk');
+  const unknownOptions = program.parseOptions(program.normalize(process.argv.slice(2))).unknown;
+
+  if (unknownOptions.length > 0) {
+    for (let i = 0, len = unknownOptions.length; i < len; ++i) {
+      // If this unknown option is a '--' option and the next option looks like it might
+      // be an argument for this option, we format it as --option=argument and pass it on.
+      if (unknownOptions[i].indexOf('--') === 0 &&
+          unknownOptions[i + 1] &&
+          unknownOptions[i + 1].indexOf('-') !== 0) {
+        runnerExecArgv.push(unknownOptions[i] + '=' + unknownOptions[++i]);
```

Comment:
You are right that posix args usually use `=` for arguments with values, but node.js is behaving really awkward here:
```
$ node --require ./test.js
>
$ node --require=./test.js
node: bad option: --require=./test.js
```
and:
```
$ node --v8-pool-size 10000000
node: bad option: --v8-pool-size
$ node --v8-pool-size=10000000
>
```
For parsing `--require` they explicitly address the next argument: https://github.com/nodejs/node/blob/291b599bba0f484f35a2deebb2350b178c1f1758/src/node.cc#L3673-L3680 but for the v8 argument, they require the equal sign: https://github.com/nodejs/node/blob/291b599bba0f484f35a2deebb2350b178c1f1758/src/node.cc#L3725

If you find an argument starting with `--` and containing a `=`, you just copy one. But if you find one with `--` but without `=`, it might be a flag (e.g. `--zero-fill-buffers`) or an argument with a value ( `--require ./test.js`). You could maintain a list with arguments that require values, but this is ugly and might change with future version...

So yeah, you are right, there is no easy solution to that. I have no idea what to do either :(